packages/office-ui-fabric-react/src/components/Facepile/Facepile.test.tsx

```diff
@@ -15,12 +15,12 @@ import { Persona, PersonaSize } from '../Persona';
 describe('Facepile', () => {
   const facepilePersonas: IFacepilePersona[] = [
     {
-      imageUrl: './images/persona-female.png',
+      imageUrl: 'https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets/persona-female.png',
```

Comment:
I don't want this embedded in there.

You could make a testImages.ts that exports consts if you really want to have a set of testImage urls.